types/mertasan__tailwindcss-variables/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for @mertasan/tailwindcss-variables 2.6
+// Project: https://github.com/mertasan/tailwindcss-variables#readme
+// Definitions by: Liam Martens <https://github.com/LiamMartens>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Config, PluginCreator } from 'tailwindcss/types/config';
+import type { PluginOptions } from './internal/PluginOptions';
+
+declare const plugin: {
+    (options: PluginOptions): { handler: PluginCreator; config?: Config };
+    __isOptionsFunction: true;
+};
+
+export = plugin;
```

Comment:
you can inline types for options here:
```suggestion
import { Config, PluginCreator } from 'tailwindcss/types/config';

declare namespace plugin {
    interface PluginOptions {
        darkToRoot?: boolean;
        variablePrefix?: string;
        colorVariables?: boolean;
        extendColors?: Record<string, string>;
        forceRGB?: boolean;
        toBase?: boolean;
    }
}

declare const plugin: {
    (options: plugin.PluginOptions): { handler: PluginCreator; config?: Config };
    __isOptionsFunction: true;
};

export = plugin;
```